src/ai/backend/manager/api/gql/scaling_group/types.py

```diff
@@ -0,0 +1,302 @@
+    )
+
+
+@strawberry.type(description="Added in 25.18.0. Network configuration for a scaling group")
+class ScalingGroupNetworkConfig:
+    wsproxy_addr: str = strawberry.field(
+        description="App-proxy coordinator API server address, Manager uses this address to communicate with the app-proxy coordinator for managing service routes, endpoint registration, and proxying client connections to session services."
+    )
+    wsproxy_api_token: str = strawberry.field(
+        description="Authentication token for the App-proxy coordinator API server."
+    )
+    use_host_network: bool = strawberry.field(
+        description="Whether session containers use the host's network namespace instead of isolated container networking. Enables direct host port binding but reduces isolation."
+    )
+
+
+@strawberry.type(description="Added in 25.18.0. Driver configuration for resource allocation")
+class ScalingGroupDriverConfig:
+    name: str = strawberry.field(
+        description="Agent resource driver implementation name. 'static' uses a predefined set of agents registered to this scaling group."
+    )
+    options: JSON = strawberry.field(
+        description="Driver-specific configuration options. Contents vary by driver type."
+    )
+
+
+@strawberry.type(description="Added in 25.18.0. Scheduler configuration options")
+class ScalingGroupSchedulerOptions:
+    allowed_session_types: list[str] = strawberry.field(
+        description="Session types that can be scheduled in this scaling group. Valid values: 'interactive' , 'batch', 'inference'. Requests for unlisted types are rejected."
+    )
+    pending_timeout: float = strawberry.field(
+        description="Maximum time in seconds a session can wait in PENDING state before automatic cancellation. Zero means no timeout. Used to prevent indefinite resource waiting when no agents are available."
+    )
+    config: JSON = strawberry.field(
+        description="Scheduler-specific configuration options. Contents depend on the scheduler implementation (fifo/lifo/drf). Used for advanced scheduling behavior customization."
+    )
+    agent_selection_strategy: str = strawberry.field(
+        description="Algorithm for selecting target agents when scheduling sessions. 'dispersed' spreads sessions across available agents, 'concentrated' packs sessions onto fewer agents, 'roundrobin' cycles through agents sequentially."
+    )
+    agent_selector_config: JSON = strawberry.field(
+        description="Configuration for the agent selection strategy. Structure varies by strategy - for example, concentrated strategy may specify endpoint spreading rules."
+    )
+    enforce_spreading_endpoint_replica: bool = strawberry.field(
+        description="When true with concentrated strategy, forces inference service replicas to be distributed across different agents instead of co-locating on same agent. Improves fault tolerance for model serving."
+    )
```

Comment:
It seems good to first explain the description of the value, and then explain what behavior occurs when it is set to true.